checkST2110.js

```diff
@@ -1401,6 +1494,83 @@ const test_22_74_1 = (sdp, params) => {
   return errors;
 };
 
+const ssnPermitted40 = ['ST2110-40:2018', 'ST2110-40:2021'];
+
+// ST 2110-40:2023 Section 7 Test 1 - Check SSN is the required fixed value
+const test_40_2023_7_1 = (sdp, params) => {
+  let [mtParams, errors] = extractMTParams(sdp, params);
+  for (let stream of mtParams) {
+    if (typeof stream.SSN !== 'undefined') {
```

Comment:
Seems the wording of ST 2110-40:2023 requires an SSN Value so might want to add an error that there is no SSN specified.

Reply:
But ST 2110-40 original revision did not define the SSN, so we need to make it clear that it would be valid for that revision to omit it.

Reply:
I will add a check that SSN is present when TN is present, i.e. in the -40:2023 case.

Reply:
Done